Source/dx.cpp

```diff
@@ -238,17 +236,14 @@ void dx_reinit()
 
 void InitPalette()
 {
-	Palette = SDL_AllocPalette(256);
-	if (Palette == nullptr) {
-		ErrSdl();
-	}
+	Palette = SDLWrap::AllocPalette(256);

```

Comment:
I'm fine with us assuming that palettes contains 256 colors to maybe simplify this.